lib/std/setutils.nim

```diff
@@ -38,6 +38,16 @@ template toSet*(iter: untyped): untyped =
 
 macro enumElementsAsSet(enm: typed): untyped = result = newNimNode(nnkCurly).add(enm.getType[1][1..^1])
 
+func emptySet*[T](U: typedesc[T]): set[T] {.inline.} =
```

Comment:
This is just `T.default` but specialized for a set.

Reply:
alternative to `default(set[T])` and similar to `fullSet` usage

Reply:
I think it's useful to newcomers which are more textual and don't know the alien syntax of sets compared to other languages.
But if you think `{}` is enough I don't have problem remove the `emptySet` and `isEmpty`

Reply:
More useful to newcomers and everybody else (!) is when you don't have ten ways of writing something. Nim is not Perl.